tests/unit/test_bruteforce.cpp

```diff
@@ -67,10 +67,10 @@ TEST_F(BruteForceTest, brute_force_vector_update_test) {
 
     // id2label size should remain the same, although we seemingly tried to exceed
     // initialcapacity.
-    ASSERT_EQ(reinterpret_cast<BruteForceIndex *>(index)->idToLabelMapping.size(), n);
+    ASSERT_EQ(reinterpret_cast<BruteForceIndex_Single *>(index)->idToLabelMapping.size(), n);
```

Comment:
consider changin the test name to brute_force_single_vector_add_test

Reply:
Why? it is an update test